docs/dsc_getting_started.md

```diff
@@ -7,22 +7,22 @@
 ### Enabling
 - WinRM will need to be enabled on the machine for the LCM to do it's job. This is due to it leveraging "implicit remoting". Documentation for WinRM can be found [here](https://docs.microsoft.com/en-us/windows/win32/winrm/installation-and-configuration-for-windows-remote-management) or you can use the quick setups with either of the following commands to enable with default configuration. These commands will not work if any network adapter is set to use the public network profile. It is possible to bypass this but not recommended for security reasons, recommended solution is to set them to domain or private. This includes virtual adapters such as ones created by VPNs or Hypver-V.
```

Comment:
```suggestion
- WinRM will need to be enabled on the machine for the LCM to do its job. This is due to it leveraging "implicit remoting". Documentation for WinRM can be found [here](https://docs.microsoft.com/en-us/windows/win32/winrm/installation-and-configuration-for-windows-remote-management), or you can use the quick setups with either of the following commands to enable it with a default configuration. These commands will not work if any network adapter is set to use the _Public_ network profile. If any network adapter does not have the "Public" network profile, the recommended solution is to set it to _Domain_ or _Private_. This includes virtual adapters such as ones created by VPNs or Hypver-V.
```

Reply:
I was able to confirm about WinRM not working with a Public network profile, but not that virtual adapters were included. Are you sure about that one?

Reply:
Yeah I ran into this during a demo with EPS actually lol. Profiles are adapter specific and my hyper-v adapter was stuck on public.

Reply:
That might be for Hyper-V only. Aside from the VPN adapter, my VMware virtual adapters don't have a network profile at all.

Reply:
Regardless, I'd add the changes I suggested for clarity.